apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/CoverageTransformerTest.java

```diff
@@ -50,20 +74,24 @@ public void transformSampleARecord() throws FHIRException {
             .filter(r -> r instanceof Beneficiary)
             .map(r -> (Beneficiary) r)
             .findFirst()
-            .get();
+            .orElseThrow();
```

Comment:
Both `.get()` and `.orElseThrow()` are identical.  Curious why you switched.  Just to make it more obvious that it would throw?

Reply:
Yeah, it's a code smell in most static analysis to use raw get without either explicitly defining that you want it to throw or doing some check before the get. Mostly a static analysis cleanup so it wont complain.